x/lockup/keeper/msg_server.go

```diff
@@ -208,8 +208,11 @@ func (server msgServer) ForceUnlock(goCtx context.Context, msg *types.MsgForceUn
 	}
 
 	// check that given lock is not superfluid staked
-	synthLocks := server.keeper.GetAllSyntheticLockupsByLockup(ctx, lock.ID)
-	if len(synthLocks) > 0 {
+	synthLock, err := server.keeper.GetSyntheticLockupByUnderlyingLockId(ctx, lock.ID)
+	if err != nil {
+		return &types.MsgForceUnlockResponse{Success: false}, errorsmod.Wrap(sdkerrors.ErrInvalidRequest, err.Error())
+	}
+	if synthLock != (types.SyntheticLock{}) {
```

Comment:
ditto for `IsNil`

Reply:
added here https://github.com/osmosis-labs/osmosis/pull/5265/commits/dd70c9e078bf96b344521dbbb4e8254e30af2ddb